Okta/OktaAppAuth.swift

```diff
@@ -58,6 +62,7 @@ public func isAuthenticated() -> Bool {
 public func clear() {
     // Clear auth state
     tokens?.clear()
+    OktaKeychain.clearAll()
```

Comment:
TokenManager already clears keychain in clear function. Probably we have to call OktaKeychain.clearAll() only when tokens object is nil?

Reply:
Agree. Removed.